plugins/regex_revalidate/regex_revalidate.c

```diff
@@ -1,524 +0,0 @@
-/** @file
```

Comment:
Did you do `git mv regex_revalidate.c regex_revalidate.cc` before you started editing?  This might have been a faster review if it was easier to see what was changed.

Reply:
perhaps.  the changes are fairly destructive regardless.